src/Machete.HL7.Tests/Segments/FT.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Machete.HL7.Tests.Segments
+{
+    using System;
+    using System.Runtime.Serialization;
+
+
+    [Serializable]
+    public class FT :
```

Comment:
You can't because it loads based on the assembly. You have to pick an assembly location.